components/form/KeyValue.vue

```diff
@@ -214,11 +214,6 @@ export default {
       type:    String,
       default: 'icon-minus',
     },
-    removeAllowed: {
-      type:    Boolean,
-      default: true,
-    },
-
```

Comment:
I think we should use an AND, as you may be allowed to edit, but you want to enforce removing capabilities, as the default value is `true`.

Cases we want to be sure to keep it disabled are on fact:
- Component can be removed but is in view mode
- Component cannot be removed but is in edit mode
- Component cannot  removed and is in view mode

It would then be only possible if the mode is not view and there's no specified restrictions.

Reply:
Great points, what you've added looks good to me.